Libraries/LibJS/Heap/GCPtr.h

```diff
@@ -190,6 +190,10 @@ class GCPtr {
 template<typename T>
 using RawGCPtr = GCPtr<T>;
 
+// Non-Owning NonnullGCPtr
+template<typename T>
+using RawNonnullGCPtr = NonnullGCPtr<T>;
```

Comment:
I added `IGNORE_GC` to Cell.h and the clang plugin for some LibWeb abuse of raw pointers/references a few weeks (months?) ago. Should we replace the use of that with this `Raw` escape hatch instead? :thinking:

Reply:
Oh hmm.. maybe `IGNORE_GC` is actually the more descriptive option!

Reply:
Looks like you added this without any uses now.